src/handlers/localstorage/nonceManager.ts

```diff
@@ -0,0 +1,29 @@
+import { getGlobal, saveGlobal } from './common';
+
+export const NONCE_MANAGER = 'nonceManager';
+
+interface NetworkNonceInfo {
+  nonce: number;
+}
+interface AccountNonceInfo {
+  [key: string]: NetworkNonceInfo;
+}
+
+export interface NonceManager {
+  [key: string]: AccountNonceInfo;
+}
+
+export interface NonceManagerUpdate {
+  network: string;
+  account: string;
+  nonce: number;
+}
+
+export const getNonceManager = async (): Promise<NonceManager> => {
+  const nonceManager = await getGlobal(NONCE_MANAGER, []);
```

Comment:
the last argument should be the default value if it doesn't exist in localstorage which is `{}` instead of `[]`

Reply:
Ahh. I followed the TS compiler too blindly here.

I think this is the first `.ts` file to consume `getGlobal`. Its definition was using a default param of `[]` for the empty state parameter, so the TS compiler was inferring a type of `any[]`.

I looked through the project and it doesn't appear that any files are relying on this default value, so I removed it and the compiler seems pleased enough now. This allowed me to properly pass in the default value of `{}`.

Let me know if you'd like a different approach.